WIP-CHANGELOG.md

```diff
@@ -54,6 +54,8 @@
   - Renamed `grafana-ops` to `grafana-dashboards`
   - Moved all grafana value templates to a separate folder
   - Moved grafana config from `prometheus.grafana` and `user.grafana` to `grafana.ops` and `grafana.user`
+- Install requirements also updates requirements
+  - It will also not try to install already correct installed requirements
```

Comment:
```suggestion
- Requirements installer will now update binaries if they are of the wrong version
  - It will also not try to install already correct installed requirements
```